### ISSUE TITLE
fix: improve accessibility of settings menu

### ISSUE THREAD

**Issue by krishnanand5** [xability/maidr#346]
The entities in the settings menu lack required aria attributes, hence making it difficult for screen readers to recognize components easily. This needs to be addressed on priority.
